404.md

```diff
@@ -0,0 +1,8 @@
+---
+layout: page
+title: 404 Not found
```

Comment:
But the title is working because is being shown

Reply:
You're using a different title. You can also skip the translate file step and in this file directly call translations, since the structure will not differ much per language (in an article style page, some pages may include extra content that is language specific) 

inside 404.md, and then you can delete the 404.html page
```markdown
---
layout: page
show_in_footer: false
---

# 404
## {% t global.404 %}
```

Reply:
You right!

Reply:
I am unable to reproduce that. I removed the title, and it still shows me `404` from the `# 404` and if I remove that, then it does not have a title (which is semantically incorrect).